langchain/src/document_loaders/web/azure_blob_storage_container.ts

```diff
@@ -0,0 +1,53 @@
+import { BlobServiceClient } from "@azure/storage-blob";
+import { AzureBlobStorageFileLoader } from "./azure_blob_storage_file.js";
+import { BaseDocumentLoader } from "../base.js";
+import { Document } from "../../document.js";
+import { UnstructuredLoaderOptions } from "../fs/unstructured.js";
+
+interface AzureBlobStorageContainerConfig {
+  connectionString: string;
+  container: string;
+}
+
+export class AzureBlobStorageContainerLoader extends BaseDocumentLoader {
+  private readonly connectionString: string;
+
+  private readonly container: string;
+
+  private readonly unstructuredLoaderOptions?: UnstructuredLoaderOptions;
+
+  constructor(
+    { connectionString, container }: AzureBlobStorageContainerConfig,
```

Comment:
My preference would be to have one config object:
```
{
  azureConfig: AzureBlobStorageContainerConfig,
  unstructuredConfig: UnstructuredLoaderOptions
}
```
to allow for easier migration for future changes